docs/modules/mssql.md

```diff
@@ -0,0 +1,180 @@
+| `firewallRules`  | `array` | Specifies the firewall rules to apply to the server. Definition follows. | N | `[]` Left unspecified, Firewall will default to only Azure IPs. If rules are provided, they must have valid values. |
+| `firewallRules[n].name` | `string` | Specifies the name of the generated firewall rule |Y | |
+| `firewallRules[n].startIPAddress` | `string` | Specifies the start of the IP range allowed by this firewall rule | Y | |
+| `firewallRules[n].endIPAddress` | `string` | Specifies the end of the IP range allowed by this firewall rule | Y | |
+
+##### Bind
+
+Creates a new user on the SQL Server. The new user will be named randomly and granted permission to log into and administer the database.
+
+###### Binding Parameters
+
+This binding operation does not support any parameters.
+
+###### Credentials
+
+Binding returns the following connection details and credentials:
+
+| Field Name | Type | Description |
+|------------|------|-------------|
+| `host` | `string` | The fully-qualified address of the SQL Server. |
+| `port` | `int` | The port number to connect to on the SQL Server. |
+| `database` | `string` | The name of the database. |
+| `username` | `string` | The name of the database user (in the form username@host). |
+| `password` | `string` | The password for the database user. |
+
+##### Unbind
+
+Drops the applicable user from the SQL Server.
+
+##### Deprovision
+
+Deletes both the database and the SQL Server instance.
+
+### Service: azure-sql-dbms-only
+
+| Plan Name | Description |
+|-----------|-------------|
+| `sql-dbms-only` | Azure SQL Server DBMS Only|
+
+#### Behaviors
+
+##### Provision
+
+Provisions a new SQL Server DBMS only. Databases can be created through subsequent provision reqquests using the `azure-sql-db-only` service.
+###### Provisioning Parameters
+
+| Parameter Name | Type | Description | Required | Default Value |
+|----------------|------|-------------|----------|---------------|
+| `location` | `string` | The Azure region in which to provision applicable resources. | Required _unless_ an administrator has configured the broker itself with a default location. | The broker's default location, if configured. |
+| `resourceGroup` | `string` | The (new or existing) resource group with which to associate new resources. | N | If an administrator has configured the broker itself with a default resource group and nonde is specified, that default will be applied, otherwise, a new resource group will be created with a UUID as its name. |
+| `tags` | `map[string]string` | Tags to be applied to new resources, specified as key/value pairs. | N | Tags (even if none are specified) are automatically supplemented with `heritage: open-service-broker-azure`. |
+| `alias` | `string` | Y | Specifies an alias that can be used by later provision actions to create databases on this DBMS. |
```

Comment:
Looks like description and required are accidentally reversed on this row.

Reply:
The same is true for MySQL.

Reply:
And PostgreSQL.

Reply:
Because I copy-pasta'ed them. Fixing!